examples/semantic_segmentation/eval_semantic_segmentation.py

```diff
@@ -52,16 +58,20 @@ def get_dataset_and_model(dataset_name, model_name, pretrained_model,
     elif model_name == 'segnet':
         model = SegNetBasic(
             n_class=n_class, pretrained_model=pretrained_model)
+    elif model_name == 'deeplab_v3plus_xception65':
+        model = DeepLabV3plusXception65(
+            pretrained_model=pretrained_model)
```

Comment:
Can you pass `n_class` and `input_size`?

Reply:
@yuyu2172 
OK,
btw, I named `crop` (corresponding to `input_size` in PSPNet) following official implementation, but it looks strange in ChiainerCV.
Should it be renamed to `input_size`?

Reply:
https://github.com/chainer/chainercv/pull/757#discussion_r257459198